src/Graphics/SpriteBatch.cs

```diff
@@ -31,6 +31,9 @@ public class SpriteBatch : GraphicsResource
 		private const int MAX_VERTICES = MAX_SPRITES * 4;
 		private const int MAX_INDICES = MAX_SPRITES * 6;
 
+		// Max array size for cpu side buffer
```

Comment:
```suggestion
		/* This is the largest array size for a VertexBuffer using VertexPositionColorTexture.
		 * note that we do NOT change the GPU buffer size since that would break XNA accuracy,
		 * but if you want to optimize your batching you can make this change in a custom SpriteBatch.
		 */
```